BUILD.md

```diff
@@ -31,6 +33,35 @@ cd seedpod_ground_risk
 pip install -r requirements.txt
 ```
 
+## Linux
+
+1. Install dependencies
+
+```bash
+sudo apt install libgeos-dev libgdal-dev proj-bin libbz2-dev
+```
+
+2. Create and activate a virtual environment, any should do. Here we use `virtualenv`:
+
+```bash
+python -m venv spgr
+source spgr
```

Comment:
I believe this should be `source spgr/bin/activate`.

Reply:
Good catch... I used the zsh alias